infrastructure/staging/dashboard.tf

```diff
@@ -157,14 +157,21 @@ module "dashboard_stageprod" {
     AzureAd__ClientId                     = "iiif-builder/common/azuread-clientid"
     Storage__ClientId                     = "iiif-builder/common/storage/clientid"
     Storage__ClientSecret                 = "iiif-builder/common/storage/clientsecret"
-    Dlcs__ApiKey                          = "iiif-builder/common/dlcs-apikey"
-    Dlcs__ApiSecret                       = "iiif-builder/common/dlcs-apisecret"
+    # the /staging-new/ api key is for Protagonist 
+    Dlcs__ApiKey    = "iiif-builder/staging-new/dlcs-apikey"
+    Dlcs__ApiSecret = "iiif-builder/staging-new/dlcs-apisecret"
   }
 
   env_vars = {
     "ASPNETCORE_ENVIRONMENT"                    = "Staging-Prod"
     "CacheInvalidation__InvalidateIIIFTopicArn" = data.aws_sns_topic.iiif_test_invalidate_cache.arn
     "CacheInvalidation__InvalidateApiTopicArn"  = data.aws_sns_topic.api_stage_invalidate_cache.arn
+    "Dlcs__SkeletonNamedQueryTemplate"          = "https://neworchestrator.dlcs.io/iiif-resource/wellcome/preview/{0}/{1}"
```

Comment:
Do all these env vars need to come from tf or can the come from the appSettings? They aren't secret and lots of other vars come from there?

Reply:
No, they don't need to come from TF - they can come from appSettings. 

I added them here for simplicity, and to avoid needing to make a PR to update appSettings as it's only a temporary change. Once we release Protagonist it'll go back to what's in appSettings now. Is the preference to have all of that in appSettings?